wire-ios/Wire-iOS/Sources/Components/Buttons/LegacyButton.swift

```diff
@@ -37,6 +37,14 @@ class Button: LegacyButton {
         applyStyle(style)
     }
 
+    convenience init(style: ButtonStyle, cornerRadius: CGFloat = 0) {
```

Comment:
If I run `Button(style: someStyle)`, which init will be used?

From line 27:
```swift
convenience init(style: ButtonStyle,
                     cornerRadius: CGFloat = 0,
                     fontSpec: FontSpec = .smallLightFont)
``` 

My suggestion is to remove the default parameters if we have multiple convince inits, so that it's clear.

Reply:
Done!